### ISSUE TITLE
can not work with Single page application

### ISSUE THREAD

**Issue by LDWX** [wkhtmltopdf/wkhtmltopdf#3920]
I have a Single page application with vue, and when I use wktmltopdf to get a pdf file, it's blank.
Accordding to my analysis it's caluse by my Single page application, that this tool can not get any
context from the web's source code. 
How can I use this tool in my Single page application, please?

The source code is below:
![image](https://user-images.githubusercontent.com/8693316/40153980-c826f4d8-59be-11e8-98b5-c66a43f68091.png)

The Single page application entrance is bleow:
![image](https://user-images.githubusercontent.com/8693316/40154015-e7c44390-59be-11e8-9e84-1860d62dc04e.png)

**Comment by morganzysman** [comment]
Have you had a change to solve this ?

**Comment by LDWX** [comment]
@morganzysman sorry I don't have. I figure out that this is casused by http request, but I don't know how to fix it.

**Comment by michael-schaefer-eu** [comment]
Try it with an absolute URL in your script tag:
`<script src="http://yourdomain.com/app.js">`

wkhtmltopdf cannot resolve relative URLs. Also for Images and CSS.

**Comment by alloylab** [comment]
@LDWX Did that resolve the issue?

**Comment by Tomsgu** [comment]
Does `--javascript-delay` help here?

**Comment by Tomsgu** [comment]
Closing as there is no feedback from the author. Please feel free to reopen if the issue still persists.

**Comment by sorry510** [comment]
Don't add css in your xx.vue like <style>css</style> , use <link> type try again

**Comment by balbatross** [comment]
https://programmer.group/html-to-pdf-graphic-report-practice.html

TLDR/Deadlink: Use Nuxt/SSR or Bundle to ES5 (haven't tested)

**Comment by xtthaop** [comment]
> Try it with an absolute URL in your script tag: `<script src="http://yourdomain.com/app.js">`
> 
> wkhtmltopdf cannot resolve relative URLs. Also for Images and CSS.

it doesn't work too.

**Comment by LDWX** [comment]
您好，您的邮件我已收到我将尽快查收并回复
